exercises/robot-simulator/lib/example.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:robot_simulator/orientation.dart';
+import 'package:robot_simulator/position.dart';
+
+class Robot {
+  Orientation orientation;
+  Position position;
+
+  Robot(this.position, this.orientation);
+
+  void move(String instructions) {
+    instructions.split("").forEach((character) {
+      if (character == "R") _rotateClockwise();
+      if (character == "L") _rotateCounterClockwise();
+      if (character == "A") _advance();
+    });
+  }
+
+  void _rotateClockwise() {
+    switch (orientation) {
+      case Orientation.NORTH:
+        orientation = Orientation.EAST;
+        break;
+      case Orientation.EAST:
+        orientation = Orientation.SOUTH;
+        break;
+      case Orientation.SOUTH:
+        orientation = Orientation.WEST;
+        break;
+      case Orientation.WEST:
+        orientation = Orientation.NORTH;
+        break;
```

Comment:
Suggestion: Add a default case for all switch cases and throw a `StateError("unexpected orientation $orientation")`.